src/OrchardCore/OrchardCore.DisplayManagement/README.md

```diff
@@ -44,6 +50,12 @@ Placement information consists of:
 		"display-type": "Detail",
 		"differentiator": "Article-MyTextField",
 
+        "match": {
```

Comment:
I don't think `match` is necessary. Every filter can be a the root of the node, like `display-type` or `differentiator`.

I would also accept a string, and array of string, of a complex object (for `matchType`). You could even simplify it by not taking `matchType` at all, as i assume we'll always want 'any'. I can't find an example that someone would want to match on having multiple parts on the content item. 

ContentType can also accept multiple values in this case, right?

Reply:
I like the idea of loosing the  "match" node in that the matches are doing the same kind of thing (i.e. filtering) as display-type and differentiator as so they would be better together.

That said I personally find it a little confusing having the filters and alterations (place,wrappers, alternates, shape) alongside each other with nothing to distinguish them but I guess it keeps it simpler.

Yes allowing string, array string or object would be a good idea as well.

> I can't find an example that someone would want to match on having multiple parts on the content item.

I haven't got a use case at the minute but it feels like there one out there somewhere :)

Reply:
Maybe confusing, but only because you are coming from O1, and as long as it's document it should not be an issue.